http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/GalaxyAppConfig.scala

```diff
@@ -10,7 +10,13 @@ final case class GalaxyAppConfig(releaseNameSuffix: String,
                                  namespaceNameSuffix: String,
                                  services: List[ServiceConfig],
                                  serviceAccount: ServiceAccountName,
-                                 uninstallKeepHistory: Boolean) {
+                                 uninstallKeepHistory: Boolean,
+                                 nfsPersistenceName: String,
+                                 postgresPersistenceName: String,
```

Comment:
can we group these postgresDisk config into a separate data type such as `PostgresDiskConfig`?